packages/devtools_app/lib/src/shared/server/_analytics_api.dart

```diff
@@ -12,7 +12,11 @@ Future<bool> isFirstRun() async {
   if (isDevToolsServerAvailable) {
     final resp = await request(apiGetDevToolsFirstRun);
     if (resp?.statusCode == 200) {
-      firstRun = json.decode(resp!.body);
+      // Additionally, package:unified_analytics will show a message if it
+      // is the first run with the package or the consent message version has
+      // been updated.
+      firstRun = json.decode(resp!.body) ||
+          await dtdManager.shouldShowAnalyticsConsentMessage();
```

Comment:
This is just for my awareness, but does `resp!.body` always equal a string of `'false'` or `'true'`?

Reply:
For this API endpoint, yes. You can look at the handler for this in server_api.dart.